[2024-04-28T21:08:52.043-0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-28T21:08:52.064-0300] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: atividade_pratica_dag.create_data_folder manual__2024-04-29T00:08:05.979845+00:00 [queued]>
[2024-04-28T21:08:52.072-0300] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: atividade_pratica_dag.create_data_folder manual__2024-04-29T00:08:05.979845+00:00 [queued]>
[2024-04-28T21:08:52.073-0300] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-28T21:08:52.093-0300] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): create_data_folder> on 2024-04-29 00:08:05.979845+00:00
[2024-04-28T21:08:52.100-0300] {standard_task_runner.py:63} INFO - Started process 11621 to run task
[2024-04-28T21:08:52.104-0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'atividade_pratica_dag', 'create_data_folder', 'manual__2024-04-29T00:08:05.979845+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/atividade_pratica_dag.py', '--cfg-path', '/tmp/tmp1o2q0l8p']
[2024-04-28T21:08:52.105-0300] {standard_task_runner.py:91} INFO - Job 27: Subtask create_data_folder
[2024-04-28T21:08:52.151-0300] {task_command.py:426} INFO - Running <TaskInstance: atividade_pratica_dag.create_data_folder manual__2024-04-29T00:08:05.979845+00:00 [running]> on host ath
[2024-04-28T21:08:52.233-0300] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='atividade_pratica_dag' AIRFLOW_CTX_TASK_ID='create_data_folder' AIRFLOW_CTX_EXECUTION_DATE='2024-04-29T00:08:05.979845+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-29T00:08:05.979845+00:00'
[2024-04-28T21:08:52.234-0300] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-28T21:08:52.235-0300] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-28T21:08:52.236-0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'mkdir -p data']
[2024-04-28T21:08:52.249-0300] {subprocess.py:86} INFO - Output:
[2024-04-28T21:08:52.253-0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-28T21:08:52.254-0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-28T21:08:52.278-0300] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=atividade_pratica_dag, task_id=create_data_folder, execution_date=20240429T000805, start_date=20240429T000852, end_date=20240429T000852
[2024-04-28T21:08:52.316-0300] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-28T21:08:52.332-0300] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-28T21:08:52.333-0300] {local_task_job_runner.py:222} INFO - ::endgroup::
